### ISSUE TITLE
VIVO-1748: Remove "RDF description of" from rdfs:label

### ISSUE THREAD

**Issue by chenejac** [vivo-project/VIVO#3337]
[Christian Hauschke](https://github.com/hauschke) (Migrated from [VIVO-1748](https://vivo-project.atlassian.net/browse/VIVO-1748)) said:

 > curl -LH'Accept: text/turtle' [https://vivo.tib.eu/fis/individual/n0000-0003-2499-7741]
> 
... or any other VIVO URI. Output:

 > <[https://vivo.tib.eu/fis/individual/n0000-0003-2499-7741/n0000-0003-2499-7741.ttl]>
>    a foaf:Document ;
>   rdfs:label "RDF description of [https://vivo.tib.eu/fis/individual/n0000-0003-2499-7741]" ;
>   dc:date "2020-03-10T08:48:47"^^xsd:dateTime ;
>   dc:publisher <[https://vivo.tib.eu/fis]> ;
>   dc:rights <[https://vivo.tib.eu/fis/termsOfUse]> .
> 
"RDF description of" should be removed as a part of rdfs:label. According to [https://www.w3.org/TR/rdf-schema/#ch_label:]

 > rdfs:label is an instance of rdf:Property that may be used to provide a human-readable version of a resource's name.
> 
Most likely the change should be done in the following files:

https://github.com/vivo-project/Vitro/blob/6d64d27fe4ad26bee7ffd10a852e71240f8ae903/api/src/main/java/edu/cornell/mannlib/vitro/webapp/controller/individual/IndividualRdfAssembler.java#L351

https://github.com/vivo-project/Vitro/blob/6d64d27fe4ad26bee7ffd10a852e71240f8ae903/api/src/test/java/edu/cornell/mannlib/vitro/webapp/controller/individual/IndividualRdfAssemblerTest.java#L305

https://github.com/vivo-project/Vitro/blob/6d64d27fe4ad26bee7ffd10a852e71240f8ae903/api/src/test/resources/edu/cornell/mannlib/vitro/webapp/controller/individual/IndividualRdfAssemblerTest.filteredRdf.n3#L150

https://github.com/vivo-project/Vitro/blob/6d64d27fe4ad26bee7ffd10a852e71240f8ae903/api/src/test/resources/edu/cornell/mannlib/vitro/webapp/controller/individual/IndividualRdfAssemblerTest.unfilteredRdf.n3#L167

Thanks to Sarven who pointed this out.

**Comment by hauschke** [comment]
Yes, rdfs:label should be the label of the individual. The provided RDF is not telling a lot in general. This could be improved, but maybe in a first step we can concentrate on providing minimal but correct information.
